internal/liteconfig/config.go

```diff
@@ -53,7 +54,8 @@ func NewDefaultConfig() (*Config, error) {
 			Level:      "debug",
 			OutputFile: "",
 		})),
-		portProvider: &portProvider{},
+		portProvider:    &portProvider{},
+		FrontendAddress: broadcastAddress,
```

Comment:
Maybe this should just default to an empty string after we rename to `FrontendIP`, since this config option is mutually exclusive with binding on `localhost` according to https://github.com/temporalio/temporal/blob/6daf3b6fb631cb58ae9522d5413b727a03531973/common/config/config.go#L89-L94

Reply:
Moved this and pulled in changes here
https://github.com/DataDog/temporalite/pull/17/files